src/pages/404.tsx

```diff
@@ -0,0 +1,24 @@
+import { NextPage } from 'next';
+
+import { ErrorNotFound } from '@shared/components';
+import { ApiService } from '@shared/services/api-service';
+
+interface NotFoundPageProps {
+	nlTranslations: Record<string, string>;
+}
+
+const NotFoundPage: NextPage<NotFoundPageProps> = ({ nlTranslations }) => {
+	return <ErrorNotFound nlTranslations={nlTranslations} />;
```

Comment:
Waarom kan je hier de `i18n` instance niet aanroepen maar op andere `NextPage`'s wel? Ook de variabelen prefix'en met `nl` is weinig schaalbaar.

Reply:
Die error pagina word gerenderd op de server en blijkbaar voor de i18n geïnitialiseerd is.

Dit is blijkbaar een gekend probleem met nextjs. 
Deze comment raad de huidige approach aan:
https://github.com/vercel/next.js/discussions/12477#discussioncomment-105212

Dit zal idd moeten herbekeken worden eens we frans toevoegen.

Reply:
Verder in beweren ze dat het [probleem is opgelost in `10.0.9`](https://github.com/vercel/next.js/discussions/12477#discussioncomment-501933), wij draaien `12.1.0`.